cmd/erasure.go

```diff
@@ -186,7 +187,7 @@ func getDisksInfo(disks []StorageAPI, endpoints []string) (disksInfo []madmin.Di
 			}
 			info, err := disks[index].DiskInfo(context.TODO())
 			di := madmin.Disk{
-				Endpoint:       endpoints[index],
+				Endpoint:       info.Endpoint,
```

Comment:
Probably let's keep endpoints[index] here.. if err is not nil, we can lose the disk endpoint here and it will be harder to know which disk that has the issue.

Reply:
@vadmeste please send the PR to this - let's merge this PR